third_party/java/java_tools/update_java_tools.sh

```diff
@@ -19,27 +19,11 @@ declare -A tool_name_to_target=(["JavaBuilder"]="src/java_tools/buildjar:JavaBui
 ["JarJar"]="third_party/jarjar:jarjar_command_deploy.jar")
 
 usage="This script updates the checked-in jars corresponding to the tools "\
-"used by the Java rules in Bazel.
+"used by the Java rules in Bazel. For details on how to upgrade the Java tools
```

Comment:
nit: Add "set -euo pipefail" ? To make sure if something fails in the script that the whole script fails.